loopchain/blockchain/vote.py

```diff
@@ -69,39 +66,38 @@ def votes(self):
     def target_hash(self):
         return self.__target_hash
 
-    @staticmethod
-    def __make_vote_init(audience):
+    def __make_vote_init(self, audience):
         if not audience:
             return None
```

Comment:
Returning `{}` is better than `None`.
There are many dangers of None exception.

Reply:
👍